code/parse.go

```diff
@@ -57,14 +60,22 @@ func parseELBlogs(request events.S3Event, getContentsFromS3Bucket GetContentFrom
 }
 
 func parseS3logs(request events.S3Event, getContentsFromS3Bucket GetContentFromS3Bucket) []ingest.Log {
+	var arn string
 	bucketName := request.Records[0].S3.Bucket.Name
 	fileName := request.Records[0].S3.Object.Key
 
 	content := getContentsFromS3Bucket(bucketName, fileName)
 
-	originBucketName := strings.Split(content, " ")[1]
+	buff := []byte(content)
+	filetype := http.DetectContentType(buff)
```

Comment:
You can do this in one line 
`filetype := http.DetectContentType([]byte(content))`

Reply:
done